weave/index/index_sqlite.py

```diff
@@ -130,9 +131,13 @@ def to_pandas_df(self, max_rows=1000, offset=0, **kwargs):
             [info[1] for info in
              self.cur.execute("PRAGMA table_info(pantry_index)").fetchall()]
         )
-        query = "SELECT * FROM pantry_index ORDER BY UUID LIMIT ? OFFSET ?"
+        query = "SELECT * FROM pantry_index ORDER BY UUID"
+        params = None
+        if max_rows:
+            query += " LIMIT ? OFFSET ?"
+            params = (max_rows,offset)
```

Comment:
Raises error if max_rows is manually set to None

Reply:
Hum. I think this one works...

Reply:
I'm still having a problem using max_rows=None when I use to_pandas_df as part of the SQLite Index.
Ie I run:
```python
pantry = weave.Pantry(weave.IndexSQLite, pantry_path="basket-data")
df = pantry.index.to_pandas_df(max_rows=None)
```
And I get the following:
```
    138     params = (max_rows,offset)
    139 ind_df = pd.DataFrame(
--> 140     self.cur.execute(query, params)
    141     .fetchall(),
    142     columns=columns
    143 )
    144 ind_df["parent_uuids"] = ind_df["parent_uuids"].apply(ast.literal_eval)
    145 ind_df["upload_time"] = pd.to_datetime(
    146     ind_df["upload_time"],
    147     unit="s",
    148     origin="unix",
    149 )

ProgrammingError: parameters are of unsupported type
```

Reply:
You're right. This has been resolved.